### PR TITLE
3 update our existing themes

### PR DESCRIPTION
**Pull request by DivyanshKhanna26**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -19,7 +19,9 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 'latest'
+          php-version: '7.4'
+        - name: Print PHP Version
+        run: php -v
```
Comment (limestreet) [inline .github/workflows/push.yml:24]: I would remove those 2 lines for now :) they have clearly wrong indentation :)

**Review by limestreet — CHANGES_REQUESTED** [review]

**Comment by limestreet** [comment]
@DivyanshKhanna26 so we have screenshot update here, right?
Looks good :)

**Comment by DivyanshKhanna26** [comment]
> @DivyanshKhanna26 so we have screenshot update here, right? Looks good :)

Yes.
I am just checking scss references in the theme. will delete it and then remove the folder.

**Comment by limestreet** [comment]
@DivyanshKhanna26 I believe we may remove the folder, there should be no issues...
However, as you suggest - there maybe some refernces which may be simply not needed anymore:
- not sure about slick.css - is it included in style.css or loaded separatelly? Probably loaded separately, so need to stay
- gruntfile - the part related to scss generation and watch directive might be not needed
- another thing comming to my mind is autoprefixer part in gruntfile  - it adds prefixes on build time... so not sure if we need that, but currently generated on built style.css may be different that the one in repository, so maybe we should copy it from build to repo and submit pr? I'm curious if this one attached in zip here -> https://github.com/fatthemes/pencil/releases/tag/v1.6.4 is different than the one in repo.

**Comment by DivyanshKhanna26** [comment]
> * another thing comming to my mind is autoprefixer part in gruntfile  - it adds prefixes on build time... so not sure if we need that, but currently generated on built style.css may be different that the one in repository, so maybe we should copy it from build to repo and submit pr
I checked the css files after they were generated and there is no difference between the generated and the one we have pushed.

Also I have removed the scss code

**Comment by limestreet** [comment]
@DivyanshKhanna26 
I see .netbeans.xml config file... I believe we may remove that too... I used netbeans years ago.

**Comment by limestreet** [comment]
@DivyanshKhanna26 and style.css.map we may probably remove as well... it should map .scss to .css generation rules, so probbaly no longer needed

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 node_modules
 nbproject
 vendor
+composer.lock
```
Comment (limestreet) [inline .gitignore:4]: @DivyanshKhanna26 missing empty line at the end :)

**Review by limestreet — CHANGES_REQUESTED** [review]
@DivyanshKhanna26 we need to bump version in Pencil, as I will need to upload it to wp repo. And there is 1.6.4 already -> https://wordpress.org/themes/pencil/
I think we may bump it to **v1.7.0** as there is quite big clean-up.

**Pull request merged**